website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +61,19 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+### On Demand Assessments
+
+-> **Note:** On Demand Assessments are only available through browser sessions. If the workspace requirements are not satisfied, you should see one or more error messages on the page indicating the blockers to enable health assessments. The button will be disabled until the requirements are satisfied.
+
+
+If you are an administrator for the workspace and all [workspace requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements) are satisfied, a **Start health assessment** button is available on the Health pages with Ready status.
+
+  ![On Demand Assessments Screenshot: Start health assessment button](/img/docs/on-demand-assessments.png)
+
+You should see a success or error message box in the left-bottom corner of the page, after clicking the button to indicate if the assessment was triggered. The button will be disabled when the assessments are in progress indicating the status. 
+Based on factors like the size of the workspace and network latency, the assessment could take several minutes.
```

Comment:
```suggestion
When you click the assessment button, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment depends can vary based on network latency and the number of resources managed by the workspace. While an assessment is an progress, you cannot trigger another assessment.
```

could you also clarify:
- can a user trigger another assessment immediately after, or if there's a cooldown period?
- does this reset the clock for automatic assessments?

Reply:
The user can trigger an assessment as soon as the "Start health assessment" button is enabled. It will be disabled until the "Assessment in progress" status is displayed alongside. So within a few minutes or more, depending on the backend queue to process assessments, users can trigger another assessment "immediately". 

No this will not affect the automatic assessments schedule.